test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -356,35 +367,7 @@ public SignatureTrustAndValidityVerificationProviderTests(SigningTestFixture fix
                 }
             }
         }
-        public async Task GetTrustResultAsync_WithNoSigningCertificate_Throws()
```

Comment:
[PR 1938](https://github.com/NuGet/NuGet.Client/pull/1938) incorrectly reintroduced a functional test which had been removed via [PR 1929](https://github.com/NuGet/NuGet.Client/pull/1929) because it's no longer possible to test this scenario using the `Signature` type.  There's already a `Signature` test for this.